packages/runtime/runtime-utils/src/requestParser.ts

```diff
@@ -13,6 +13,9 @@ export class RequestParser implements IRequest {
      * @param url - the url to get path parts of
      */
     public static getPathParts(url: string): readonly string[] {
+        if (url.split("/")[0] === "" && url.split("/").length === 2) {
```

Comment:
That's pretty expensive piece of code! Parsing and allocating array twice, just to do trivial test?
Also not sure why /a is special here. i.e. /a -> [a], but /a/b -> ["", a, b].
Don't we want to fix seconds case as well?

Reply:
I just deleted this part and I will push later